### ISSUE TITLE
[FEATURE REQUEST] Proxy support for EXR multipart output

### ISSUE THREAD

**Issue by jreichel-nvidia** [AcademySoftwareFoundation/OpenImageIO#4263]
It looks as if EXR multipart output does not support I/O proxies. In my experiments all methods seem to succeed, but I never receive any data from my proxy. There is a comment in exroutput.cpp:
```
    // m_output_multipart.reset (new Imf::MultiPartOutputFile (*m_output_stream,
    //                                          &m_headers[0], subimages)();
    // FIXME: Oops, looks like OpenEXR 2.0 currently lacks a
    // MultiPartOutputFile ctr that takes an OStream, so we can't
    // do this quite yet.
```
Maybe I'm missing something but it looks as if that constructor is available since at least OpenEXR 2.0.0:
```
    MultiPartOutputFile (
        OStream&      os,
        const Header* headers,
        int           parts,
        bool          overrideSharedAttributes = false,
        int           numThreads               = globalThreadCount ());
```
Isn't that the one that is needed?

**Comment by lgritz** [comment]
Was closed by #4263

**Comment by lgritz** [comment]
> Does this fix meet the requirements for a merge to the 2.5 branch?

@jreichel-nvidia Yes, certainly, it will be in the June 1 patch release.
